### ISSUE TITLE
More variables

### ISSUE THREAD

**Issue by danieljharris** [tanhakabir/rest-book#103]
It would be amazing if we could manually set variables in addition to assigning them from request responses.

Set variables something like this:
```
let url = "https://www.myapi.com";

let port = 3000;

let default_header = User-Agent: rest-book;

let name = "Foo";
```

And use them in later cells (maybe with better syntax):
```
POST ($url)($port)/api
$default_header
Content-Type: application/json

{
    name: $name,
    text: "Foo is the most bar of the Foos" 
}
```

And be able to reuse them throughout:
```
GET ($url)($port)/users
$default_header
```

**Comment by nat1craft** [comment]
I second this! More ~~cowbell~~ Variables!   Highly useful.

**Comment by tanhakabir** [comment]
sure! can do

**Comment by tanhakabir** [comment]
Add ability to use constants like this in release 2.0.2:

![image](https://user-images.githubusercontent.com/12758612/122684380-ee2ccb00-d1b9-11eb-99d5-8fce457046a8.png)
![image](https://user-images.githubusercontent.com/12758612/122684568-f5a0a400-d1ba-11eb-8a19-3f50fbbfe932.png)


I'll publish 2.0.2 on the marketplace once I get better reception

**Comment by nat1craft** [comment]
I noticed that using the variable as the endpoint for a POST (for example) did not seem to work.  If I however copy/pasted the value in place of the variable, it works fine.

![image](https://user-images.githubusercontent.com/6864673/123159220-c0a28480-d43a-11eb-98cd-c3afea56ae11.png)

**Comment by tanhakabir** [comment]
@nat1craft fixed in 2.2.0!

**Comment by nat1craft** [comment]
Confirmed; That is a really nice feature.  Thanks again for your hard work!
